### ISSUE TITLE
GSuite mirror

### ISSUE THREAD

**Issue by Kirens** [dtekcth/dfotose#23]
In the name of backups it might be interesting to investigate whether there are any sweet API:s to have uploads also go to the divisions GSuite Drive.

Maybe this is a functionality better suited in a separate backup-script?

**Comment by cpluss** [comment]
Might be better to use a Amazon S3 Bucket for this, as it's a lot of data if we're to back up every image.

**Comment by emilhem** [comment]
Maybe even Amazon Glacier for a really low cost backup

**Comment by Rembane** [comment]
Should we start by buying a couple of external HDDs and running backup on them? It will be cheaper and faster to get started than by using an external service.

**Comment by Kirens** [comment]
Yeah, I've been thinking of Glaicer for quite some time, wondered if the GSuite solution would easier and quicker to get up and running. But it seems to be a bad alternative.

For the second question, Do you think the backup part belongs to this project, or should it be handled externally?
My thoughts are inconclusive:
 * [pro] There are plenty cache-files, (i.e. Thumbnails) that this project hosts, the organization of those needs to be coordinated between the two projects.
 * [con] We might as well backup the thumbnails as well.
 * [con] Coupling the backup script to this projects makes changes to backups more complicated.
 * [pro] This is the main project we maintain in need of proper backups.
 * [pro] I understand a lot of references and naming is stored in MongoDB and that might be easier to take care of within the project
